### ISSUE TITLE
Power issue

### ISSUE THREAD

**Issue by xGhost169** [aidancbrady/Mekanism#2643]
My power doesn't come trough Tesseract to ultimate cables, to fix it I must replace the cables...
Cables just stop working after chunks get unloaded, no power goes into those. It might be same with mechanical pipes.

**Comment by Wishbone1977** [comment]
I'm experiencing this as well, although some tesseracts still work, but only ones that were placed before the issue arose. It seems to have occurred in one of the recent Resonant Rise 3 updates (that's the modpack I'm running).

All in all, I'm having some extremely odd problems with my Mekanism power network.

1. My induction matrix outputs no power. It's a 4x4x4 matrix (including the casing) with 7 basic cells and 1 advanced provider. It's got 8 ports on 2 sides, 4 on each. One side is connected to my power plant, and I've tried drawing power from the other side, but to no avail. It's filling up with power from my power plant just fine, but I can't get any power back out of it.

2. Since I exchanged my 8 ultimate energy cubes with the induction matrix, no power has been going into my tesseract power network, obviously, since no power comes out of the induction matrix. I've even removed the tesseract connected to the induction matrix. However, I have at least one network being fed power from a tesseract on that frequency. It appears that infinite power is somehow being generated out of thin air.
